reference-implementation/lib/abstract-ops/writable-streams.js

```diff
@@ -657,6 +659,9 @@ function WritableStreamDefaultControllerErrorIfNeeded(controller, error) {
 }
 
 function WritableStreamDefaultControllerGetBackpressure(controller) {
+  if (controller._releaseBackpressure === true) {
```

Comment:
We might have to add `&& controller._queue.length === 0` here. Not sure yet. 🤔

Reply:
This check is indeed necessary. It's covered by this new test: https://github.com/web-platform-tests/wpt/pull/31742/commits/f69d4b733da5cba396b95b7379bffb4f352aacb1